include/RAJA/pattern/params/reducer.hpp

```diff
@@ -5,11 +5,11 @@
 #include "RAJA/util/SoAPtr.hpp"
 
 #if defined(RAJA_CUDA_ACTIVE)
-#define DEVICE cuda
 #include "RAJA/policy/cuda/MemUtils_CUDA.hpp"
+using device_mem_pool_t = RAJA::cuda::device_mempool_type;
 #elif defined(RAJA_HIP_ACTIVE)
-#define DEVICE hip
 #include "RAJA/policy/hip/MemUtils_HIP.hpp"
+using device_mem_pool_t = RAJA::hip::device_mempool_type;
```

Comment:
I didn't realize this before but this should be in the RAJA detail namespace, not in the global namespace.

Reply:
@artv3 Do you have time to fix this today too?